search/indexing_api.py

```diff
@@ -109,6 +109,9 @@
     "image_src": {"type": "keyword"},
     "topics": {"type": "keyword"},
     "offered_by": {"type": "keyword"},
+    "created_on": {"type": "date"},
+    "default_search_priority": {"type": "integer"},
+    "minimum_price": {"type": "float"},
```

Comment:
Can we use a `scaled_float` here instead? It would help avoid rounding errors. https://stackoverflow.com/questions/30767089/how-to-store-money-in-elasticsearch

Reply:
Done. I updated the price in the runs serializer to use `scaled_float` as well so the two have the same format